src/Factbox/Factbox.php

```diff
@@ -100,6 +105,17 @@ public function useInPreview( $preview ) {
 	}
 
 	/**
+	 * @note contains information about action=info
+	 *
+	 * @since 2.3
+	 *
+	 * @param boolean
+	 */
+	public function useInActionInfo( $actionInfo ) {
```

Comment:
The format of this method is suspect - it is probably better to have `->foo()` and `->notFoo()` than `->foo( true )` and `->foo( false )`. The naming is also not clear. Without investigating, I cannot tell what this does. Does this mean "is going to be used in 'action info' and should behave as such", "was used in 'action info' and should hold that into account" or "should only be used in 'action info'"?

I see you did not start this pattern though...